_episodes/11-practice-teaching.md

```diff
@@ -228,13 +229,107 @@ may have a fixed mindset about teaching. By strengthening your growth mindset wi
+  with more intensive or specialized instruction while Teacher B
+  delivers a general lesson to the main group.
+
+- **Teach and observe:** Teacher A teaches while Teacher B observes
+  the students, collecting data on their understanding to help plan
+  future lessons.
+
+- **Parallel teaching:** The class is divided into two equal groups
+  and the teachers present the same material simultaneously to each.
+
+- **Station teaching:** The students are divided into several small
+  groups that rotate from one station or activity to the next while
+  both teachers supervise where needed.
+
+All of these models are better than teaching alone because they create
+opportunities for lateral knowledge transfer; we use team teaching,
+teach and assist, and alternative teaching most often in programming
+workshops.  Team teaching is particularly beneficial in day-long
+workshops: not only does it give each teacher's voice a chance to
+rest, it reduces the risk that they will be so tired by the end of the
+day that they will start snapping at their students or fumbling at
+their keyboard.
+
+> Many people who aren't comfortable teaching are still willing and
+> able to provide in-class technical support.  They can help learners
+> with setup and installation, answer technical questions during
+> exercises, monitor the room to spot people who may need help, or
+> keep an eye on the shared notes and either answer questions there or
+> remind the instructor to do so during breaks.
+>
+> Helpers are sometimes people training to become teachers (i.e.,
+> they're Teacher B in the teach and assist model), but they can also
+> be members of the host institution's technical support staff,
+> alumni, or advanced learners who already know the material well.
+> Using the latter as helpers is doubly effective: not only are they
+> more likely to understand the problems their peers are having, it
+> also stops them from getting bored.
+>
+> The [helper checklist][helpers] describes what helpers can do to
+> help instructors in more detail.
+
+If you and a partner are co-teaching, try to follow these rules:
+
+- Take 2-3 minutes before the start of each class to confirm who's
+  teaching what with your partner.  (If you have time to do some
+  advance preparation, try drawing a concept map together.)
+
+- Use that time to work out a couple of hand signals as well.  "You're
+  going too fast", "speak up", "that learner needs help", and, "It's
+  time for a bathroom break" are all useful.
+
+- Each person should teach for at least 10-15 minutes at a stretch,
+  since students may be distracted by more frequent interleaving.
+
+- The person who isn't teaching shouldn't interrupt, offer
+  corrections, elaborations, or amusing personal anecdotes, or do
+  anything else to distract from what the person teaching at the time
+  is doing or saying.  The one exception is that it's sometimes
+  helpful to ask leading questions, particularly if the learners seem
```

Comment:
Non-native English speakers may not know what a leading question is. Maybe use "helpful to ask questions that may be in the minds of several learners" (not sure if that is a good way to phrase it)

Reply:
Wording changed.